apps/vscode/vscode.py

```diff
@@ -12,6 +12,8 @@
 and app.bundle: com.microsoft.VSCodeInsiders
 os: mac
 and app.bundle: com.visualstudio.code.oss
+os: mac
+and app.bundle: com.todesktop.230313mzl4w4u92
```

Comment:
this bundle id is surprising, but [appears to be correct and stable](https://forum.cursor.sh/t/cursor-bundle-identifier/779/2)